service/src/main/java/org/broadinstitute/listener/config/ListenerProperties.java

```diff
@@ -10,6 +10,16 @@ public class ListenerProperties {
   private String relayConnectionString;
```

Comment:
This class could probably be converted to a record; you can have records populated by `@ConfigurationProperties`.